local-testing/index.html

```diff
@@ -1,97 +1,48 @@
-                                            style="width:161px;">
-                                    </td>
-                                    <td style="padding-left: 10px;">
-                                        <input type="submit" style="padding-left: 10px;" class="submit" id="startChat"
-                                            value="Start Chat"></input>
-                                    </td>
-                                </tr>
-                            </tbody>
-                        </table>
-                    </div>
-                </fieldset>
-            </form>
-        </section>
         <section class="section-chat" id="section-chat" style="display: none; float: right; width: 50%;">
             <div id="root"></div>
         </section>
-    </div>
-    <script src="backendEndpoints.js"></script>
+    </body>
     <script>
-        $(function () {
+        // Initiate the widget on specified element
+        (function () {
             connect.ChatInterface.init({
-                containerId: 'root' // This is the id of the container where you want the widget to reside
-            });
-
-            $('#contactDetails').submit(function (e) {
-                e.preventDefault();
-
-                customerName = $('#firstName').val();
-                if (!customerName) {
-                    alert('you must enter a name & username');
-                    document.getElementById("contactDetails").reset();
-                } else {
-                    console.log("this is the first name:" + customerName);
-                    document.getElementById("contactDetails").reset();
-
-                    connect.ChatInterface.initiateChat({
-                        name: customerName,
-                        region,
-                        apiGatewayEndpoint,
-                        contactAttributes: JSON.stringify({
-                            "customerName": customerName
-                        }),
-                        contactFlowId,
-                        instanceId
-                    },successHandler, failureHandler);
-
-                }
-            });
-        });
-
-        function successHandler(chatSession) {
-            console.log("success!");
-            $('#section-chat').fadeIn(400);
-
-            chatSession.onChatDisconnected(function(data) {
-                $('#section-chat').hide('slide');
+                containerId: "root", // This is the id of the container where you want the widget to reside
+                shouldShowMessageReceipts: true // defualt: false
             });
-        }
-
-        function failureHandler(error) {
-            console.log("There was an error: ");
-            console.log(error);
+        })();
+
+        // Handle user chat initiation
+        function initiateChat() {
+            $("#btn").prop("disabled", true);
+            
+            // Enter custom name here
+            initConfig.name = "John";
```

Comment:
Line 42 and 43 can be removed because the name and contactAttributes can also be updated in `config.js`. We anyways need to fill in the region, endpoint and other details in that file